packages/ckeditor5-dev-changelog/bin/generate-changelog.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+import { exampleFunction } from '../dist/index.js';
```

Comment:
```
import { exampleFunction } from '../dist/index.js';
         ^^^^^^^^^^^^^^^
SyntaxError: The requested module '../dist/index.js' does not provide an export named 'exampleFunction'
```

Reply:
Fixed in [725a690](https://github.com/ckeditor/ckeditor5-dev/pull/1084/commits/725a69016251ba9fccf07724db34e5a06a75321c).

Reply:
This file is being removed.